src/crcspeed.c

```diff
@@ -104,45 +122,135 @@ void crcspeed16big_init(crcfn16 fn, uint16_t big_table[8][256]) {
     }
 }
 
+/* Note: doing all of our crc/next modifications *before* the crc table
+ * references is an absolute speedup on all CPUs tested. So... keep these
+ * macros separate.
+ */
+
+#define DO_8_1(I)                                       \
+    crc##I ^= *(uint64_t *)next##I;                     \
+    next##I += 8
+
+#define DO_8_2(I)                                       \
+    crc##I = little_table[7][(uint8_t)crc##I] ^         \
+             little_table[6][(uint8_t)(crc##I >> 8)] ^  \
+             little_table[5][(uint8_t)(crc##I >> 16)] ^ \
+             little_table[4][(uint8_t)(crc##I >> 24)] ^ \
+             little_table[3][(uint8_t)(crc##I >> 32)] ^ \
+             little_table[2][(uint8_t)(crc##I >> 40)] ^ \
+             little_table[1][(uint8_t)(crc##I >> 48)] ^ \
+             little_table[0][crc##I >> 56]
+
+#define CRC64_SPLIT(div) \
+    olen = len; \
+    next2 = next1 + ((len / div) & CRC64_LEN_MASK); \
+    len = (next2 - next1)
+
+#define MERGE_CRC(I) \
+    crc1 = crc64_combine(crc1, crc##I, next2 - next1, CRC64_REVERSED_POLY, 64)
```

Comment:
```suggestion
#define MERGE_CRC(crc_i) \
    crc1 = crc64_combine(crc1, (crc_i), next2 - next1, CRC64_REVERSED_POLY, 64)
```

I don't know if there was some method to these naming conventions, but suggesting a couple of things that would make it easy for me to read.

Reply:
I'll be switching to functions, so this shouldn't be an issue anymore.

Reply:
Fixed.